packages/cli/README.md

```diff
@@ -113,12 +113,14 @@ Set `BUBBLEWRAP_KEYSTORE_PASSWORD` for the key store password and `BUBBLEWRAP_KE
 Usage:
 
 ```
-bubblewrap build [--skipPwaValidation] [--skipSigning] [--manifest="<path-twa-manifest>"]
+bubblewrap build [--skipPwaValidation] [--skipSigning] [--manifest="<path-twa-manifest>"] [--signingKeyPath="<path-to-signing-key>"] [--signingKeyAlias="<key-alias>"]
 ```
 
 Options: 
   - `--skipPwaValidation`: skips validating the wrapped PWA against the Quality Criteria.
-  - `--skipSigning`: skips signing the built APK and App Bundle.
+  - `--skipSigning`: skips signing the built APK and App Bundle. Incompatible with `--signingKeyPath` and `--signingKeyAlias`.
+  - `--signingKeyPath`: path to keystore to use for signing the built APK and App Bundle. Overrides signingKey.path specified in `twa-manifest.json`. Incompatible with `--skipSigning`.
```

Comment:
I think potentially we're overcommunicating here. I can't think of an example where a command line parameter to a tool is overridden by the value in the config file - the parameter always takes precedence. Also, personally I'd take the "incompatible with `--skipSigning` as implied.

I'd be fine with just:

`--signingKeyPath`: path to keystore to use for signing the built APK and App Bundle.

And assume the user can figure out the other two parts.

Reply:
What I understand is that when `--skipSigning` is used, `--signingKeyPath` and `--signingKeyAlias` are ignored. I think incompatible makes it look like command will not work if those are used together, which is not the case? 

I agree that the user can probably figure out implications of `--skipSigning` , but it doesn't hurt to be explicit about it.  I'd just say other parameters related to signing are ignored when this is used. I wouldn't add extra information on this on `--signingKeyPath` and only have it on `--skipSigning`.